cmd/rook/discover.go

```diff
@@ -17,22 +17,32 @@ package main
 
 import (
 	"fmt"
+	"time"
 
 	rook "github.com/rook/rook/cmd/rook/rook"
 	"github.com/rook/rook/pkg/clusterd"
 	"github.com/rook/rook/pkg/daemon/discover"
 	"github.com/rook/rook/pkg/operator/k8sutil"
 	"github.com/rook/rook/pkg/util/exec"
+	"github.com/rook/rook/pkg/util/flags"
 	"github.com/spf13/cobra"
 )
 
-var discoverCmd = &cobra.Command{
-	Use:    "discover",
-	Short:  "Discover devices",
-	Hidden: true,
-}
+var (
+	discoverCmd = &cobra.Command{
+		Use:    "discover",
+		Short:  "Discover devices",
+		Hidden: true,
+	}
+
+	// interval between discovering devices
+	discoverDevicesIntervalSeconds time.Duration
+)
 
 func init() {
+	discoverCmd.Flags().DurationVar(&discoverDevicesIntervalSeconds, "discover-interval", 60*time.Minute, "interval between discovering devices in seconds")
```

Comment:
the variable is called *Seconds, but we give the default here in minutes and it's actually a time.Duration value type?  What exactly can be passed for this argument, a time duration string such as "60m" or "30s"? If so, the variable name and comment here should be updated too because they are implying its an integer value in the units of seconds.

Reply:
good catch, I started out with seconds, then changed it to a duration to be similar to the mon timeout in the operator.yaml. will update